cognite_toolkit/cdf_tk/templates/_migration.yaml

```diff
@@ -0,0 +1,110 @@
+- version: 0.1.0b7
+  cognite_modules: {}
+  resources: {}
+  tool:
+    - title: New required file '_system.yaml'.
+      steps:
+        - "Create a new file '_system.yaml' in the 'cognite_modules' folder."
+        - "Copy the content from 'cognite_modules/default.packages.yaml' into '_system.yaml'."
+        - "Remove the comments. This file should not be edited by a user."
+        - "Add the following comment at the top of the file: '# DO NOT EDIT THIS FILE!'."
+        - "Add the following line to the end of the file: 'cdf_toolkit_version: ${VERSION}'"
+        - "The resulting `_system.yaml` should look like this:"
+        - "<Panel>
+            # DO NOT EDIT THIS FILE!\n
+            # This is the system configuration for the Cognite Toolkit.\n
+            packages:\n
+            \u00A0  data_model_examples:\n
+            \u00A0    - cdf_auth_readwrite_all\n
+            \u00A0    - cdf_oid_example_data\n
+            \u00A0    - cdf_apm_simple_data_model\n
+            \u00A0  cdf_demo_infield:\n
+                ...\n
+            # This part is used by cdf-toolkit to keep track of the version and help you upgrade.\n
+            cdf_toolkit_version: ${VERSION}"
+    - title: "New format for configs: `config.<env>.yaml`"
+      steps:
+        - "For each entry in `environments.yaml` create a new file `config.<env>.yaml`."
+        - Copy the content from `config.yaml` into each of the new files under the key `modules`.
+        - At the top of the file, copy in the content of the entry in the `environments.yaml` file.
+          under the key `environment` in the `config.<env>.yaml` files.
+        - In the `environments` entry rename `deploy` to `selected_modules_and_packages`.
+        - In the `environments` entry add the key `name` with the name of the environment.
+        - "For example, the resulting `config.dev.yaml` should look like this:"
+        - "<Panel>
+          environment:\n
+          \u00A0  name: dev\n
+          \u00A0  project: <my-project>\n
+          \u00A0  type: dev\n
+          \u00A0  selected_modules_and_packages:\n
+          \u00A0  - cdf_functions_dummy\n
+          \u00A0  common_function_code: ./common_function_code\n
+          \n
+          modules:\n
+          \u00A0 cognite_modules:\n
+          \u00A0 \u00A0    # Globally defined variables are available to all submodules\n
+          \u00A0\u00A0    # of the cognite_modules.\n
+          \u00A0\u00A0     cdf_cluster: ${CDF_CLUSTER}\n
+           \u00A0\u00A0    cicd_clientId: ${IDP_CLIENT_ID}\n
+            ...\n
+          \u00A0  custom_module:\n
+          \u00A0\u00A0    custom_submodule:\n
+          \u00A0\u00A0\u00A0      custom_key: custom_value"
+    - title: "The file `environments.yaml` is no longer used"
+      steps:
+        - Remove it from your project.
```

Comment:
Maybe a comment on how the content is moved to config.<env>.yaml files? I assume the order if the change items will be kept, so that you have already created your new config files?